src/schemas/json/minecraft-recipe.json

```diff
@@ -322,6 +349,28 @@
         }
       }
     },
+    "commonRecipeCategory": {
+      "type": "object",
```

Comment:
Optional fix: add `title` and `description` properties.

Reply:
I'm not against the idea, but the tagsCommonToAllRecipes definition has no title or description either

Reply:
Then it's a documentation problem I guess. In such case we can nothing to do I think.

Reply:
I do a change. Tell me what you think about it.

Reply:
According to the current CONTRIBUTING.md it's incorrect: [`title` can't contain upper case letters](https://github.com/SchemaStore/schemastore/blob/master/CONTRIBUTING.md#best-practices).
Note: I'll fix it.